cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -114,6 +114,12 @@ object EvaluatorCommand : CommandWithHelp() {
 
         val evaluatorRun by lazy { evaluator.run(script) }
 
+        if (log.isErrorEnabled) {
```

Comment:
Commit message nit: Either "Otherwise the message ... is hidden" or "Otherwise the messages ... are hidden" (even if it spans multiple lines ans refers to multiple formats it could be regarded as a single message).

Reply:
Done.